README.md

```diff
@@ -9,24 +10,148 @@ relation.
+            // The middle loop runs n times
+            for (var j = 0; j < n; j++) {
+                // The inner loop runs n^2 times
+                
+                for (var k = 0; k < n * n; k++) {
+                    
                     count = count + 1;
                 }
             }
         }
+
+        
         mystery(n / 3);
     }
 }
 ```
+The function makes three recursive calls to itself with the argument being $n/3$, $3T(n/3)$ the front "3" that is multiplying the $T(n/3)$ is there to account for the three recursive calls. The function also does a triple nested loop.
+So, the outer and iner loops each run $n^2$ times and then the middle loop runs $n$ times.
+that gives you $n^2 * n * n^2 = n^5$. Together you get the relation $T(n)=3T(n/3) + n^5$. This is my recurrance realtion. Because the nested loops dominate the time complexity the tight bound for Big $O$ is $\theta (n^5)$
+
+I used the recurrsion tree method to solve the reccurance relation:
+```
+
+The function makes 3 recursive calls to itself with the argument being n/3
+
+Level 0: n^5
+|
+|--- Level 1: n^5 / 3^5
+|    |
+|    |--- Level 2: n^5 / 3^10
+|    |    |
+|    |    |--- Level 3: n^5 / 3^15
+|    |    |
+|    |    |--- Level 3: n^5 / 3^15
+|    |    |
+|    |    |--- Level 3: n^5 / 3^15
+|    |
+|    |--- Level 2: n^5 / 3^10
+|    |    |
+|    |    |--- Level 3: n^5 / 3^15
+|    |    |
+|    |    |--- Level 3: n^5 / 3^15
+|    |    |
+|    |    |--- Level 3: n^5 / 3^15
+|    |
+|    |--- Level 2: n^5 / 3^10
+|         |
+|         |--- Level 3: n^5 / 3^15
+|         |
+|         |--- Level 3: n^5 / 3^15
+|         |
+|         |--- Level 3: n^5 / 3^15
+|
+|--- Level 1: n^5 / 3^5
+|    |
+|    |--- Level 2: n^5 / 3^10
+|    |    |
+|    |    |--- Level 3: n^5 / 3^15
+|    |    |
+|    |    |--- Level 3: n^5 / 3^15
+|    |    |
+|    |    |--- Level 3: n^5 / 3^15
+|    |
+|    |--- Level 2: n^5 / 3^10
+|    |    |
+|    |    |--- Level 3: n^5 / 3^15
+|    |    |
+|    |    |--- Level 3: n^5 / 3^15
+|    |    |
+|    |    |--- Level 3: n^5 / 3^15
+|    |
+|    |--- Level 2: n^5 / 3^10
+|         |
+|         |--- Level 3: n^5 / 3^15
+|         |
+|         |--- Level 3: n^5 / 3^15
+|         |
+|         |--- Level 3: n^5 / 3^15
+|
+|--- Level 1: n^5 / 3^5
+     |
+     |--- Level 2: n^5 / 3^10
+     |    |
+     |    |--- Level 3: n^5 / 3^15
+     |    |
+     |    |--- Level 3: n^5 / 3^15
+     |    |
+     |    |--- Level 3: n^5 / 3^15
+     |
+     |--- Level 2: n^5 / 3^10
+     |    |
+     |    |--- Level 3: n^5 / 3^15
+     |    |
+     |    |--- Level 3: n^5 / 3^15
+     |    |
+     |    |--- Level 3: n^5 / 3^15
+     |
+     |--- Level 2: n^5 / 3^10
+          |
+          |--- Level 3: n^5 / 3^15
+          |
+          |--- Level 3: n^5 / 3^15
+          |
+          |--- Level 3: n^5 / 3^15
+
+Level 0: n^5 Level 1: 3 * (n/3)^5 = n^5 / 3^5
+Level 2: 9 * (n/9)^5 = n^5 / 3^10
+Level 3: 27 * (n/27)^5 = n^5 / 3^15
```

Comment:
...and you have to sum up all of those levels, you can't just take the last level.